python/ray/_private/gcs_utils.py

```diff
@@ -577,3 +577,36 @@ def use_gcs_for_bootstrap():
     This function is included for the purposes of backwards compatibility.
     """
     return True
+
+
+def cleanup_redis_storage(host: str, port: int, password: str, use_ssl: bool, storage_namespace: str):
+    """This function is used to cleanup the storage. Before we having
+    a good design for storage backend, it can be used to delete the old
+    data. It support redis cluster and non cluster mode.
+
+    Args:
+       host: The host address of the Redis.
+       port: The port of the Redis.
+       password: The password of the Redis.
+       use_ssl: Whether to encrypt the connection.
+       storage_namespace: The namespace of the storage to be deleted.
+    """
+
+    from ray._raylet import del_key_from_storage
```

Comment:
is the name `del_key_from_storage` accurate? this should clear all keys in the given namespace, right?

Reply:
it is delete a key. we use hash key. so delete the key of the hash table.

Reply:
I see, can you add a comment explaining this? It was not obvious to me

Reply:
sure, i'll.